plugins/processors/awsappsignals/internal/resolver/attributesresolver.go

```diff
@@ -31,9 +31,10 @@ type attributesResolver struct {
 
 // create a new attributes resolver
 func NewAttributesResolver(resolvers []appsignalsconfig.Resolver, logger *zap.Logger) *attributesResolver {
+	//TODO: Logic for native k8s needs to be implemented
 	subResolvers := []subResolver{}
 	for _, resolver := range resolvers {
-		if resolver.Platform == appsignalsconfig.PlatformEKS {
+		if resolver.Platform == appsignalsconfig.PlatformEKS || resolver.Platform == appsignalsconfig.PlatformK8s {
 			subResolvers = append(subResolvers, getEksResolver(logger), newEKSHostedInAttributeResolver(resolver.Name))
```

Comment:
np - this function can be renamed since this is not exclusive to just EKS, but handles k8s as well

Reply:
Ack. I refactored the internal eks.go file to kubernetes.go so this name change would be consistent